packages/react-charting/src/components/AreaChart/__snapshots__/AreaChart.test.tsx.snap

```diff
@@ -239,7 +239,7 @@ exports[`AreaChart snapShot testing renders Areachart correctly 1`] = `
                 role="none"
               >
                 <button
-                  aria-label="metaData1"
+                  aria-label="metaData1 selected"
```

Comment:
It's generally an antipattern to put information like "selected", "pressed", etc. in the accessible name for a variety of reasons. Since this button has `role="option"`, the place to define this information would be in the `aria-selected` attribute.

Right now, this markup will result in multiple screen readers announcing this as some variation of "metaData1 selected, not selected"